pynldas2/pynldas2.py

```diff
@@ -192,11 +230,21 @@ def separate_snow(clm: DF, t_rain: float = T_RAIN, t_snow: float = T_SNOW) -> DF
     return _snow_point(clm, t_rain + 273.15, t_snow)
 
 
-def _txt2df(txt: str, resp_id: int, kwds: list[dict[str, dict[str, str]]]) -> pd.Series:
+def _txt2df(
+    txt: str,
+    resp_id: int,
+    kwds: list[dict[str, dict[str, str]]],
+    model: str = "NLDAS_FORA0125_H.002",
```

Comment:
Please change `model` to `source`, since this is what we are setting here. Also, instead of having `NLDAS_FORA0125_H.002` which is difficult to memorize, use `grib` and `netcdf` as acceptable options. Please use the same for other functions.